cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -82,6 +86,8 @@ def finish_flow_cell(
             LOG.warning(f"Flow cell {flow_cell_directory_name} will be skipped: {e}")
             return
 
+        self.delete_flow_cell_data(flow_cell)
```

Comment:
Shouldn't this be after `store_flow_cell_data`?

Reply:
The idea here is to clean before we add. @Vince-janv could there be a way to do the commit in one? What if the store fails here?

Reply:
For statusDB yes, but for Housekeeper no... Right now I don't commit to status_DB before adding the new data. Maybe I should for consistency between the two databases?

Reply:
It feels sketchy, but perhaps that could be a separate PR? This was the function previously as well

Reply:
Agreed that it is sketchy. Fixing it would require a re-write of the whole HouseKeeperAPI though 😬